scripts/build/build_node/Platform/Linux/package-list.ubuntu-bionic.txt

```diff
@@ -2,8 +2,9 @@
 
 # Build Tools Packages
 cmake/3.20.1-0kitware1ubuntu18.04.1     # For cmake
-clang-12                                # For Ninja Build System
-ninja-build                             # For the compiler and its dependencies
+clang-12                                # For the compiler and its dependencies
+ninja-build                             # For Ninja Build System
+llvm                                    # For llvm-strip
```

Comment:
Is there a min version of llvm needed?

Reply:
There are specific versions, but for ubuntu 20.04, specifying llvm will install llvm-10 and create the appropriate version symlink. If you specify a particular version, it will not create the symlink, (like what it does with clang). So we want the general llvm package (which is fine for this scenario anyways)

Reply:
Is there a reason the GNU strip command can't be used?
llvm-strip is meant to be a drop in replacement and GNU strip is already installed: https://llvm.org/docs/CommandGuide/llvm-strip.html#description

Reply:
Thats probably better to use if its already there